test/multiverse/suites/rails/Envfile

```diff
@@ -3,6 +3,7 @@
 # frozen_string_literal: true
 
 RAILS_VERSIONS = [
+  ["github: 'rails'", 3.0], # Rails Edge
```

Comment:
love this, thank you

Reply:
I'm glad to hear it! I was super pleased when I realized that the previous work we did on Envfile helpers made this as easy as this.